src/test/resources/s3proxy.conf

```diff
@@ -1,6 +1,6 @@
-s3proxy.endpoint=https://127.0.0.1:0
+s3proxy.endpoint=http://127.0.0.1:0
```

Comment:
When I leave `https` I get `com.amazonaws.SdkClientException: Unable to execute HTTP request: Unrecognized SSL message, plaintext connection?`. What I am doing wrong. Even without my changes it fails with s3proxy:1.7.0.